package.json

```diff
@@ -27,19 +27,16 @@
     "@babel/runtime": "^7.4.3",
     "axios": "^0.18.0",
     "babel-loader": "^8.0.0",
-    "css-loader": "^1.0.0",
+    "css-loader": "^2.1.1",
```

Comment:
@AndrewGibson27 I ended up adding this back since vue-loader was barking about it being a missing dependency. I wanted to play it safe, but if you want to remove later, go for it!